packages/Path-View-Core.package/SScrollPane.class/instance/getUnScaledCenter.st

```diff
@@ -0,0 +1,4 @@
+zooming
+getUnScaledCenter
```

Comment:
@AlexImmer this could be a method of Nico, which got added by the zoom-slider merge. but could you refactor/rename this to #getUnscaledCenter? :)